test/NuGet.Core.Tests/NuGet.Common.Test/XmlUtilityTests.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    public class XmlUtilityTests
+    {
+        [Fact]
+        public void XmlUtility_Load_NullFilePathParameter_Throws()
```

Comment:
The type name (`XmlUtility`) is already in the test class name (`XmlUtilityTests`).

Name your test methods:  `<MemberUnderTest>_<Condition>_<Expectation>`

This test would be:  `Load_WhenFilePathIsNull_Throws`